code/modules/surgery/surgery.dm

```diff
@@ -226,8 +226,8 @@ proc/do_surgery(mob/living/carbon/M, mob/living/user, obj/item/tool, var/surgery
 				return TRUE
 
 			if(QUALITY_WELDING)
-				if(try_surgery_step(/datum/surgery_step/robotic/fix_brute, user, tool))
-					return TRUE
+				try_surgery_step(/datum/surgery_step/robotic/fix_brute, user, tool)
+				return TRUE
```

Comment:
Good fix , however i advise not reverting this , since it prevents old surgery steps from being checked , regardless if the fix brute is possible or not.

Reply:
Sure, readded it